### PR TITLE
Add a bunch of tests

### PR DESCRIPTION
**Pull request by chrismwendt**

### CODE REVIEW

File: package/src/handler.test.ts [hunk 1]
```diff
@@ -74,3 +80,228 @@ describe('search requests', () => {
...
+            pass
+        `,
+                definitionLine: 1,
+                commentStyle: pythonStyle,
+            }),
+            'docstring'
+        )
+    })
+
+    it('finds multi-line python doc block', async () => {
```
Comment (lguychard) [inline package/src/handler.test.ts]: All of these tests are `async` functions, but the test body seems sync. Am I missing something?

File: package/src/handler.ts [hunk 2]
```diff
@@ -329,6 +329,131 @@ export function referencesQueries({
...
+    }
+
+    function findDocstringInBlockComment({
+        block: { startRegex, lineNoiseRegex, endRegex },
+        lines,
+    }: {
+        block: BlockCommentStyle
+        lines: string[]
+    }): string[] | undefined {
+        // ⚠️ Local mutation
```
Comment (lguychard) [inline package/src/handler.ts]: Why is the local mutation needed?

File: package/src/handler.test.ts (hunk 1 above)
Comment (chrismwendt) [reply package/src/handler.test.ts]: I copy-pasted, so no you're not missing anything 😄

File: package/src/handler.ts (hunk 2 above)
Comment (chrismwendt) [reply package/src/handler.ts]: IMO, it turned out to be easier to understand by writing imperative steps on a mutable array of lines.

**Pull request merged**